doc-content/enterprise-only/cases/case-management-creating-hardware-spec-ready-proc.adoc

```diff
@@ -0,0 +1,17 @@
+[id='case-management-create-hardware-spec-milestone-proc']
+= Creating the Hardware spec ready milestone
+Create a *Hardware spec ready* milestone that is reached when the required hardware specification document is completed.
+
+.Procedure
+. In the process designer, expand *Milestone* in the *Object Library* and drag a new milestone on the canvas and place it on the right side of the *Place order* end event.
+. Click the new milestone and in the upper-right corner, click image:cases/left-arrows.png[] to open the *Properties* panel.
+. Input `Hardware spec ready` in the *Name* field.
+. Select `true` from the *Adhoc autostart* menu.
+. Click the down arrow in the *Assignments* field and add the following new conditions in the *Hardware spec ready Data I/O* window.
++
+Add `CaseData(data.get("hwSpec") != null)` by selecting *Constant* from the *Source* menu.
```

Comment:
s/Add `CaseData(data.get("hwSpec") != null)` by selecting *Constant* from the *Source* menu./Select *Constant* from the *Source* menu and add `CaseData(data.get("hwSpec") != null)`.

This would be more consistent with surrounding sections.